vuepress/docs/next/docs/reference/local-tips-and-tricks.md

```diff
@@ -63,23 +63,21 @@ The namespace will be recreated, preserving the images already pulled, so it's u
 
 ## Network Issues
-A local Entando 7 quickstart installation (e.g. what you'll get if you follow the [Getting Started](../../docs/getting-started/) guide) may use a set of local domain names to enable access to Entando services. Your IP address will vary, but may look something like this:
+A local Entando 7 quickstart installation (e.g. what you get if you follow the [Getting Started](../../docs/getting-started/) guide) may use a set of local domain names to enable access to Entando services. Your IP address will vary, but may look something like this:
 ```
-quickstart-entando.192.168.99.1.nip.io
-quickstart-kc-entando.192.168.99.1.nip.io
-quickstart-eci-entando.192.168.99.1.nip.io
+quickstart.192.168.99.1.nip.io
+YOUR-APP.192.168.99.1.nip.io
 ```
-The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
+The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your entando-app.yaml) borrows a fixed IP address that is created during the initial installation. This domain is used to generate ingress routes to map incoming URLs to individual services. In production environments, there's generally a dedicated network layer to manage IPs/routing (both on premise and in the cloud), but this is not readily available in most local setups. Below are a couple of common issues that can prevent Entando from initializing in a local environment:
 
 ### `.nip.io isn't allowed`
- - This could be due to firewall settings or corporate security policies. The simplest workaround is to manually edit your /etc/hosts file and map the necessary domains to the IP of your local virtual machine.
+ - This could be due to firewall settings or corporate security policies. The simplest workaround is to manually edit your /etc/hosts file and map the necessary domain to the IP of your local virtual machine.
 ```
- 192.168.99.1 quickstart-kc-entando.192.168.99.1.nip.io
- 192.168.99.1 quickstart-eci-entando.192.168.99.1.nip.io
- 192.168.99.1 quickstart-entando.192.168.99.1.nip.io
+ 192.168.99.1 quickstart.192.168.99.1.nip.io
+ 192.168.99.1 YOUR-APP.192.168.99.1.nip.io
 ```
 - If you add microservices to your installation, you may need to add additional mappings for the new ingresses.
-- See [this section below](#option-2-manually-update-your-hosts-file) for detailed steps to perform this on Windows.
+- To update files on a Windows machine, see the [Windows process](#option-2-manually-update-your-hosts-file) below.
```

Comment:
on Windows (nix machine - that gets into tricky territory with emulators, etc. safer to confine references to the OS layer.)

Reply:
Agreed, just say "on Windows"